notifications/broadcast.py

```diff
@@ -183,14 +180,13 @@ def ssl_cert_display_fields(cert_instance, noti_instance):
     noti_rule_msg = json.loads(noti_instance.rule_msg)
     relationship = str(noti_rule_msg["relationship"])
     subject_type = \
-    relationship.replace("\r", "").replace("\n", "").replace("\t", "").lower()
+    relationship.replace(" ", "").replace("\r", "").replace("\n", "").replace("\t", "").lower()
```

Comment:
please explain this line of code. i absolutely don't understand why you need it. and please try to avoid using line continuations right after = signs. this one has a much nicer formatting when splitting like

```python
'some'.replace('o', 'a').\
'same'.replace('a', 'o')
```

Reply:
relationship tells us what kind of email message has to broad cast. Currently we are dealing with 3 kinds of certificate rules
1.Has Expired
2. Will Expired within x days
3. Invalid certificate
I stripped out all white line characters from relationship value and check for first 10 character in a select switch manner